src/websockets/ohlc_handler.js

```diff
@@ -29,34 +29,24 @@ define(['websockets/binary_websockets','charts/chartingRequestMap','jquery','jqu
 
     liveapi.events.on('candles', function (data) {
         var key = (data.echo_req.ticks_history + data.echo_req.granularity).toUpperCase();
-        for ( var index in data.candles ) {
-            //This is for microsoft products. 
-            //They some how also include the include() while iterating over the object.
-            if(index === 'includes'){
-              return;
-            }
-            var eachData = data.candles[index],
-                    open  = parseFloat(eachData.open),
-                    high  = parseFloat(eachData.high),
-                    low   = parseFloat(eachData.low),
-                    close = parseFloat(eachData.close),
-                    time  = parseInt(eachData.epoch) * 1000;
+        data.candles.forEach(function(eachData){
```

Comment:
Yes I have tested it on all the major browsers including IE and Edge.

For the second one there is one possibility that the returned array could be empty, which `forEach` handles pretty well.